Source/JavaScriptCore/runtime/StringPrototypeInlines.h

```diff
@@ -1327,7 +1327,12 @@ ALWAYS_INLINE JSString* replaceUsingRegExpSearch(VM& vm, JSGlobalObject* globalO
             if (!sourceRanges.tryConstructAndAppend(lastIndex, result.start)) [[unlikely]]
                 OUT_OF_MEMORY(globalObject, scope);
 
-            cachedCall.clearArguments();
+            if (!cachedCall) {
```

Comment:
Should this be marked unlikely? Although, maybe it's best left to PGO in this case.